code/game/objects/items/devices/scanners.dm

```diff
@@ -202,7 +202,7 @@ REAGENT SCANNER
 //		user.show_message("\blue Bloodstream Analysis located [M.reagents:get_reagent_amount("epinephrine")] units of rejuvenation chemicals.")
 	if (M.has_brain_worms())
 		user.show_message("\red Subject suffering from aberrant brain activity. Recommend further scanning.")

```

Comment:
i only changed the internal organs path names and the cyber implant detectection... I AM NOT takking out the color macro code for the entire file that i only touched two things on. No. ;-;